### ISSUE TITLE
Copy as many referenced images as possible to the docs repo

### ISSUE THREAD

**Issue by mglukhovsky** [rethinkdb/docs#813]
Some images referenced on the docs repo (e.g. http://rethinkdb.com/docs/paas/) aren't included in this repository. For example, this image isn't included: `/assets/images/posts/2014-10-14-compose-newdep.png`

This means the standalone docs don't have access to those images. Could we make a concrete list of images used in the documentation that don't belong to this repo?

**Comment by chipotle** [comment]
There are three occasions of this, based on a search for "/images/[^doc]" in the docs repository.

```
rethinkdb-vs-mongodb.md, line 98: /assets/images/screenshots/data_explorer_cropped.png
paas.md, line 24: /assets/images/posts/2014-10-14-compose-newdep.png
index.html, line 34: /assets/images/videos/what-is-rethinkdb.png
```

The last one is the landing page and a placeholder for a video, so it's probably not relevant here.

For the other two, if there are better places to copy or move them (I don't think the screenshot is actually referenced anywhere else and I'm not sure why I put it there), let me know where better places are and I'll move them -- the ontology under /images/docs/ is a little opaque to me.

**Comment by chipotle** [comment]
This was actually fixed in pull request #837, so I'll close this corresponding issue.
